typescript/web/src/utils/tests/data.fixtures.ts

```diff
@@ -71,9 +177,83 @@ export const DEEP_DATASET_WITH_CLASSES_DATA: DatasetData = {
       },
     },
   ],
+  images: [],
+};
+
+const NESTED_LABEL_CLASS_IN_DATASET_WITH_LABELS_DATA: Omit<
```

Comment:
Maybe that it's something that should rather been done in the next "test-cleanup PR" but I feel like with these things we're going too far away from the idea to have something reusable everywhere.
Can't we move such cases (relations data) in the query mocks instead?

Reply:
Yeah, now it would probably make more sense to stop having relations in `data.fixtures.ts` (and maybe provide some helper function to create relations, I will see if it seems useful or not).